test/Microsoft.OpenApi.Tests/Attributes/DisplayAttributeTests.cs

```diff
@@ -14,6 +15,17 @@ public enum ApiLevel
         Corporate = 3
     }
 
+    [Flags]
+    public enum UserType
+    {
+        [DisplayAttribute("admin")]
+        Admin = 1,
+        [DisplayAttribute("editor")]
+        Editor = 2,
+        [DisplayAttribute("publisher")]
+        Publisher = 3
+    }
```

Comment:
(nit) It might be nice to also test:

```suggestion
        [DisplayAttribute("all")]
        All = Admin | Editor | Publisher
    }
```